pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1562,6 +1562,15 @@ type RemoteWriteSpec struct {
 	// Whether to enable HTTP2.
 	// +optional
 	EnableHttp2 *bool `json:"enableHTTP2,omitempty"`
+
+	// When enabled, remote-write will resolve the URL host name via DNS, choose one of the IP addresses at random, and connect to it.
```

Comment:
```suggestion
    // When enabled:
    //     - The remote-write mechanism will resolve the hostname via DNS.
    //     - It will randomly select one of the resolved IP addresses and connect to it.
    //
    // When disabled (default behavior):
    //     - The Go standard library will handle hostname resolution.
    //     - It will attempt connections to each resolved IP address sequentially.
    //
    // Note: The connection timeout applies to the entire resolution and connection process. 
    //       If disabled, the timeout is distributed across all connection attempts.
    //
    // Requires Prometheus v3.1.0 or later.
    //
    // +optional
```

Reply:
Thanks @nicolastakashi 
Update doc done